README.md

```diff
@@ -182,64 +191,134 @@ require("jupynium").setup({
 
-### Open and attach to a Jupynium server
+In Neovim, with your Jupynium `.ju.py` file open, you can run `:JupyniumStartAndAttachToServer` to start the notebook server.
 
-**This is for local neovim only. For remote neovim, see [Command-Line Usage](#%EF%B8%8F-command-line-usage-attach-to-remote-neovim).**
+#### Sync current buffer to the Jupynium server
 
-Running `:JupyniumStartAndAttachToServer` will open the notebook.  
-Type password and once **you need to be on the main page (file browser) for the next steps**.
+You need to be on the main notebook page (file browser) for the next few steps.
 
-**New in 0.1.1:**  
-Jupynium will open Jupyter Notebook server for you if not found.  
-It will also open the ipynb file in the current directory and ask you if you want to sync from vim or from ipynb.
+Although Neovim is now attached to the server, it won't automatically start syncing.
 
-### Sync current buffer to the Jupynium server
-
-You attached your nvim instance to the server, but it won't automatically start syncing.  
-Run `:JupyniumStartSync` to create a new ipynb file and start syncing from vim to notebook.
+To sync your Neovim Jupynium file to a notebook, run `:JupyniumStartSync`.
 
 You can also:
 
-1. `:JupyniumStartSync filename` to give name (`filename.ipynb`) instead of `Untitled.ipynb`.
+- `:JupyniumStartSync filename` to give a name to the notebook (`filename.ipynb`) instead of `Untitled.ipynb`. This does not open existing files. If a file with that name already exists then the filename argument will just be ignored.
+- To sync a Jupynium file to an existing notebook, manually open the file in the browser, and `:JupyniumStartSync 2` to sync to the 2nd tab (count from 1).
 
-- This will not open a file if it exists. Instead filename will be ignored.
+At this point, any changes you make within the Neovim Jupynium file will be reflected live in the browser. Make sure you do not make changes inside the browser itself, as the sync is only one-way (from Neovim to browser).
 
-2. Manually open the file from the browser, and `:JupyniumStartSync 2` to sync using the 2nd tab (count from 1).
+If you want to save a copy of the `.ipynb` file, run `:JupyniumSaveIpynb`. There is also a configuration option to enable automatic saving/downloading.
 
-### Use multiple buffers
+#### Sync multiple Jupynium files
 
-Run `:JupyniumStartSync` again with a new buffer you want to sync.
+You can sync multiple files at the same time. Simple run `:JupyniumStartSync` again with the new file you want to sync.
 
-### Use multiple neovim
+#### Use multiple Neovim
 
-You can run `:JupyniumStartSync` on a new neovim instance.  
+You can run `:JupyniumStartSync` on a new Neovim instance.  
 If you have `auto_attach_to_server = false` during setup, you need to run `:JupyniumAttachToServer` and `:JupyniumStartSync`.
 
-## 📝 .ju.py (or .ju.\*) file format
+## 👨‍💻️ Command-Line Usage (attach to remote Neovim)
```

Comment:
Is there any reason the command line usage is in this section? Maybe it's more closely related, but this is just for advanced use cases and users shouldn't need to know for most cases, that's why I put it at the near end. I think the file format may need to come first or emphasise in this section that if you don't plan to do so just skip to the next section?